### PR TITLE
Dev guide

### PR DESCRIPTION
**Pull request by nilshempelmann**
Writing test to be completed

### CODE REVIEW

File: docs/source/dev_guide.rst [hunk 1]
```diff
@@ -147,7 +140,24 @@ its own Sphinx documentation and is referenced by the main birdhouse document. P
...
+ .. _writing_tests:
+
+ Writing tests
+ -------------
+
+ Writing test is an essential part of software development. The WPS templates produced by Cookiecutter_ are already including the initial folders for units tests and basic dependencies in the environment.
+ There are two parts of tests:
+
+ * Unit tests:
+ python nose-tests to check the functionality of functions and processes. They are stored in the folder `{bird WPS}/tests` and appropriate test data  `{bird WPS}/tests/testdata`.
```
Comment (tlvu) [inline docs/source/dev_guide.rst:154]: From memory, I think it's pytest and not nose-tests.  Do we use nose in other birds?

Comment (huard) [reply docs/source/dev_guide.rst:154]: I don't remember seeing nose anywhere.

**Comment by tlvu** [comment]
Does this PR fixes https://github.com/bird-house/birdhouse-docs/issues/46?  Might want to tag the issue in the description to auto-close the issue.

File: docs/source/dev_guide.rst (hunk 1 above)
Comment (nilshempelmann) [reply docs/source/dev_guide.rst]: changed to pytest

File: docs/source/dev_guide.rst [hunk 2]
```diff
@@ -147,7 +140,24 @@ its own Sphinx documentation and is referenced by the main birdhouse document. P
...
    The directive will clone and checkout the repository, then include these external files as if
-   they were part of the native documentation. 
+   they were part of the native documentation.
+
+ .. _writing_tests:
+
+ Writing tests
+ -------------
+
+ Writing test is an essential part of software development. The WPS templates produced by Cookiecutter_ are already including the initial folders for units tests and basic dependencies in the environment.
```
Comment (Zeitsperre) [inline docs/source/dev_guide.rst:150]:
```suggestion
 Writing tests is an essential part of software development. The WPS templates produced by Cookiecutter_ include the initial folders needed for units tests and basic dependencies in the environment.
```

**Pull request merged**